rewrite-maven/src/main/java/org/openrewrite/maven/ChangeParentPom.java

```diff
@@ -165,6 +165,7 @@ public Validated validate() {
                                         doAfterVisit(new ChangeTagValueVisitor<>(tag.getChild("version").get(), newVersion));
                                     }
                                     doAfterVisit(new RemoveRedundantDependencyVersions());
+                                    maybeUpdateModel();
```

Comment:
Note: There is a `maybeUpdateModel()` in `visitDocument()` that won't be necessary if our recipes call `maybeUpdateModel()` after updates.